pkg/api/v1beta3/dynakube/activegate/props.go

```diff
@@ -93,9 +93,7 @@ func (ag *Spec) IsMetricsIngestEnabled() bool {
 }
 
 func (ag *Spec) NeedsService() bool {
```

Comment:
why keep the check at all? if there's an activegate there should be a service

Reply:
Ok, will adapt it so it is always created

Reply:
Removed it